sample_consensus/include/pcl/sample_consensus/impl/sac_model_line.hpp

```diff
@@ -221,7 +221,12 @@ pcl::SampleConsensusModelLine<PointT>::optimizeModelCoefficients (
 
   // Compute the 3x3 covariance matrix
   Eigen::Vector4f centroid;
-  compute3DCentroid (*input_, inliers, centroid);
+  if (0 == compute3DCentroid (*input_, inliers, centroid))
```

Comment:
Any particular reason for using `0 == compute()`? A simple `if (!compute())` will also work

Reply:
I think compute3DCentroid counts the number of points used, so comparing with 0 gives a clearer understanding of what is returned - you suggestion makes an implicit conversion to bool?